FluentAssertions.Core/Equivalency/EquivalencyAssertionOptionsBase.cs

```diff
@@ -67,18 +72,55 @@ protected EquivalencyAssertionOptionsBase(IEquivalencyAssertionOptions defaults)
             includeProperties = defaults.IncludeProperties;
             includeFields = defaults.IncludeFields;
 
-            selectionRules.AddRange(defaults.SelectionRules);
+            selectionRules.AddRange(defaults.SelectionRules.Where(IsNotStandardSelectionRule));
+
+            if (IncludesIncludingSelectionRule(defaults.SelectionRules))
```

Comment:
I do not think there is a practical way for this code to be hit at this point in time.  It more is an attempt to future proof for the global confgiuration